test/src/main/java/org/apache/accumulo/test/AuditMessageIT.java

```diff
@@ -468,7 +468,7 @@ public void testDeniedAudits() throws AccumuloSecurityException, AccumuloExcepti
       auditAccumuloClient.tableOperations().offline(OLD_TEST_TABLE_NAME);
     } catch (AccumuloSecurityException ex) {}
     try (Scanner scanner = auditAccumuloClient.createScanner(OLD_TEST_TABLE_NAME, auths)) {
-      scanner.iterator().next().getKey();
+      var unusedRetVal = scanner.iterator().next().getKey();
     } catch (RuntimeException ex) {}
```

Comment:
It looks like this is a pattern where we are expecting the error. This should be refactored to use `assertThrows` instead of trying to ignore the return value.

Reply:
Re-worked  quite a few instances within AuditMessageIT to use assertThrows. Think I found all code portions that could use assertThrows regardless of whether it affected EP error patterns.